CIP-SUBLEDGER/README.md

```diff
@@ -0,0 +1,144 @@
+---
+CIP: ?
+Title: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+Status: Proposed
+Category: Ledger
+Authors:
+    - Micah Kendall <micah@butane.dev>
+Implementors: [Micah Kendall, <you?>]
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-05-30
+License: CC-BY-4.0
+---
+
+# CIP \*\*\*: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+
```

Comment:
```suggestion
```
There is no H1 element in the CIP outline since this is provided by the title in the YAML header.